src/erc20/filters/titler.ts

```diff
@@ -0,0 +1,28 @@
+import { IEventTitle, TitlerFilter } from '../../interfaces';
+import { EventKind } from '../types';
+
+/**
+ * This a titler function, not to be confused with the labeler -- it takes a particular
+ * kind of event, and returns a "plain english" description of that type. This is used
+ * on the client to present a list of subscriptions that a user might want to subscribe to.
+ */
+export const Title: TitlerFilter = (kind: EventKind): IEventTitle => {
+  switch (kind) {
+    case EventKind.Approval:
+      return {
+        title: 'Delegation Approved',
+        description: 'One account delegated tokens to another..',
```

Comment:
We should update the copy here -- this gets displayed at the "subscriptions" page -- but currently that page isn't active, so it's not a big deal.

Reply:
Update it to what?

Reply:
Maybe just remove the double period... we can deal with updating the actual copy later.